app/presenters/linked_item_presenter.rb

```diff
@@ -22,6 +22,11 @@ def present
       presented[attr.to_s] = linked_item.send(attr) if linked_item.has_attribute?(attr)
     end
 
+    linked_topic = linked_item.linked_topic
```

Comment:
Not a fan of assignments inside conditionals. I think it makes the code more clever, which I try to avoid.